assets/javascripts/discourse/components/realtime-validation-settings.js.es6

```diff
@@ -0,0 +1,35 @@
+import Component from "@ember/component";
```

Comment:
@fzngagan Please use the existing naming convention for wizard admin components, i.e. ``wizard-thing``. e.g. ``wizard-realtime-validation``.

Reply:
Done.